### PR TITLE
Sidebar and todo

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
- Added Sidebar component
- Added todo feature

### CODE REVIEW

File: extension-app/src/components/Todo.jsx [hunk 1]
```diff
@@ -1,7 +1,74 @@
...
+            placeholder="Add a task and press Enter"
+          />
+          <ul className=" text-light text-left flex flex-col-reverse">
+            {localTodos.map((taskObj) => (
+              <li className="list-item flex justify-content-space-bet">
+                <span>{taskObj.task}</span>
+                <input
+                  type="checkbox"
+                  checked={taskObj.isDone}
+                  value={taskObj.task}
```
Comment (harshalini) [inline extension-app/src/components/Todo.jsx:46]: You can destructure isDone and task from taskObj

File: extension-app/src/components/Todo.jsx [hunk 2]
```diff
@@ -1,7 +1,74 @@
-import React from 'react'
+import { useEffect, useState } from "react";
+import { FiTrash2 } from "react-icons/fi";
+import { v4 as uuid } from "uuid";
+import { useLocalStorage } from "../utils/hooks/useLocalStorage";
+
+const Todo = () => {
+  const [localTodos, setLocalTodos] = useLocalStorage("todos", []);
```
Comment (harshalini) [inline extension-app/src/components/Todo.jsx:7]: Liked that you created hook for localStorage

**Review by harshalini — APPROVED** [review]
Clean code and naming conventions are also good

File: extension-app/src/components/Sidebar.jsx [hunk 3]
```diff
@@ -1,7 +1,148 @@
...
+      {showSidebar && (
+        <aside className="sidebar-container br-m px-xs py-s">
+          <div>
+            <div className="flex justify-content-space-bet">
+              <p className="text-light font-xbold mx-xs">Wallpaper</p>
+              <button className="btn btn-outline" onClick={resetHandler}>
+                Reset
+              </button>
+            </div>
+            {/* Wallpaper settings */}
```
Comment (nakulsharma15) [inline extension-app/src/components/Sidebar.jsx:69]: You can remove commented code after it's no longer needed

File: extension-app/src/components/components.css [hunk 4]
```diff
@@ -47,6 +61,31 @@
     outline: transparent;
     color: var(--text-light);
   }
+
+/* todo */
```
Comment (nakulsharma15) [inline extension-app/src/components/components.css:65]: You can remove commented code if it's not needed

File: extension-app/src/components/Todo.jsx (hunk 1 above)
Comment (vaibhav-mahalle) [reply extension-app/src/components/Todo.jsx:46]: Thank you for pointing out will keep that in mind.

**Review by nakulsharma15 — APPROVED** [review]
Suggested a few minor changes. Apart from that, everything looks good. I Liked the naming of functions , CSS Classes, and the overall folder structure.

File: extension-app/src/components/Sidebar.jsx (hunk 3 above)
Comment (vaibhav-mahalle) [reply extension-app/src/components/Sidebar.jsx:69]: will keep that in mind.

File: extension-app/src/components/components.css (hunk 4 above)
Comment (vaibhav-mahalle) [reply extension-app/src/components/components.css:65]: It helps in segregating the code but will look into it.

**Pull request merged**